maps/perseus.navcfg

```diff
@@ -0,0 +1 @@
+autojump 0.2
```

Comment:
Can we get a comment like "bot gets stuck in X room with autojump 0.5"? You can use C/C++-style comments.